### DISCUSSION TITLE
Version 1.17.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#2902] [Changelog]
## Bugfixes
- [#2852](https://github.com/ScratchAddons/ScratchAddons/pull/2852) gamepad and debugger do not show correctly in RTL. (prio-4)
- [#2852](https://github.com/ScratchAddons/ScratchAddons/pull/2852) mouse-pos causes clone counter to move constantly. (prio-4)
- [#2874](https://github.com/ScratchAddons/ScratchAddons/pull/2874) scratchr2 does not add border radius to profile banners. (prio-4)
- [#2871](https://github.com/ScratchAddons/ScratchAddons/pull/2871) cloud-games uses wrong color. (prio-4)
- [#2879](https://github.com/ScratchAddons/ScratchAddons/pull/2879) infinite-scrolling does not load more if a page is already scrolled. (prio-4)
- [#2884](https://github.com/ScratchAddons/ScratchAddons/pull/2884) debugger uses wrong color for logged procedure arguments. (prio-4)
- [#2909](https://github.com/ScratchAddons/ScratchAddons/pull/2909) Addons change order after enabling in search result. (prio-3)
- [#2949](https://github.com/ScratchAddons/ScratchAddons/pull/2949) Expanding addon entry in settings is hard. (prio-3)
- [#2961](https://github.com/ScratchAddons/ScratchAddons/pull/2961) Warnings are hard to see. (prio-4)
- [#2977](https://github.com/ScratchAddons/ScratchAddons/pull/2977) scratch-messaging crashes when fetching Scratch Team messages fail. (prio-3)
- [#2979](https://github.com/ScratchAddons/ScratchAddons/pull/2979) copy-message-link crashes when the user is muted. (prio-3)
- [#2980](https://github.com/ScratchAddons/ScratchAddons/pull/2980) infinite-scroll lists duplicate items on studios. (prio-3)
- [#2985](https://github.com/ScratchAddons/ScratchAddons/pull/2985) studio-tools sometimes does not work. (prio-3)
- [#3016](https://github.com/ScratchAddons/ScratchAddons/pull/3016) studio-tools uses undefined string. (prio-2)

## New Features
- [#2525](https://github.com/ScratchAddons/ScratchAddons/pull/2525) New addon: better-img-uploads (prio-3)
- [#2761](https://github.com/ScratchAddons/ScratchAddons/pull/2761) editor-searchable-dropdowns: allow creating variables, lists or messages in the dropdown (prio-3)
- [#2818](https://github.com/ScratchAddons/ScratchAddons/pull/2818) New addon: disable-stage-drag-select (prio-3)
- [#2879](https://github.com/ScratchAddons/ScratchAddons/pull/2879) infinite-scrolling: add a page separator on forums (prio-3)
- [#2884](https://github.com/ScratchAddons/ScratchAddons/pull/2884) debugger: allow translating blocks (prio-3)
- [#2884](https://github.com/ScratchAddons/ScratchAddons/pull/2884) debugger: support "add block by name" in editor-devtools (prio-3)
- [#2894](https://github.com/ScratchAddons/ScratchAddons/pull/2894) onion-skinning: allow dynamic disabling and enabling (prio-4)
- [#2947](https://github.com/ScratchAddons/ScratchAddons/pull/2947) New addon: old-studio-layout (prio-3)
- [#2955](https://github.com/ScratchAddons/ScratchAddons/pull/2955) Allow dynamic disabling and enabling on many addons. (prio-4)
- [#2963](https://github.com/ScratchAddons/ScratchAddons/pull/2963) expanding-search-bar: allow dynamic disabling and enabling (prio-4)
- [#2967](https://github.com/ScratchAddons/ScratchAddons/pull/2967) editor-stepping: Allow changing highlight color (prio-3)
- [#2975](https://github.com/ScratchAddons/ScratchAddons/pull/2975) New addon: studio-followers (prio-3)

**Comment by ItsEnderium** [comment]
now its catching up with minecraft 1.17 and fortnite's 17th season.

**Comment by Shluffy** [comment]
#2884 is on twice for new features

**Reply by cobaltt7** [reply]
intentional
